src/REST/Section/Project.php

```diff
@@ -31,6 +31,20 @@ public function get($project, array $expand = [])
         return $this->Jira->get("project/{$project}", $parameters);
     }
 
+    /**
+     * Returns all projects which are visible for the currently logged in user. If no user is logged in, it returns the list of projects that are visible when using anonymous access.
```

Comment:
PSR-2 recommends lines to be under 120 characters, but this was not required. In Badoo, we have the same rule which is also not required
BUT, to be honest there is nothing that prevents you from splitting your line into two here. AND this makes code much better looking

Reply:
Got it